### ISSUE TITLE
Redudancy in stateset

### ISSUE THREAD

**Issue by ipendlet** [darkreactions/ESCALATE_Capture#29]
From Scott Novotney
"A data questio for you. In crank 0021, I see 465426 unique rows in `0021.stateset.csv`. But there are only 59,670 unique triplets of the `_rxn_M_{inorganic,organic,acid}`
So theree are many redundant triplets — are these different from one another somehow?
the rows in the csv appear to be identical"

```grep 0021 0021.stateset.csv  | grep -v '#' | cut -d, -f 3- | sort | uniq | wc
   59670   59670 92192632```
Thats the command I ran:
• grep for `0021` to remove the header
• remove comments
• print columns 3+ (ignoring the `dataset,name` column
• sort
• only print unique
• count up the lines/characters

That takes awhile to do the sort. You can just look at the three reactions which is a bit faster to grep.
```grep 0021 0021.stateset.csv  | grep -v '#' | cut -d, -f 3-5 | sort | uniq  |wc```

**Comment by ipendlet** [comment]
This version update does not change the associated labels between experiments in the state space and the actual being performed, but updates the actual concentrations of acid and inorganic being given to models in the stateset.  Specifically, the bug found in escalate state set generation created the following problem:

1) The second addition of any chemical (in this case acid, and organic) was not being correctly summed with the first addition prior to the calculation of the concentration.  
2) This resulted in inaccurate presentation of the formula concentration to models using the state set

In addition to re-uploading the statesets we have reached out to consumers / stakeholders in the escalation/escalate to resolve any issues that this mislabeling has caused in their dataset.  We are working with @snovot and @hamed to rerun predictions and baselines on the test harness.

**Comment by ipendlet** [comment]
*Tldr*: A recent bug in the `stateset` generation was recently brought to our attention and the relevant updates have been pushed.  This bug does not affect the main perovskite dataset.  All performers who were affected have been contacted and we are working on resolutions on an individual basis.


We identified a bug present in the code used to generate the `stateset`.  The bug affected the `statesets` associated with crank0014 - crank0022.  The issue did not affect the perovskite dataset to the best of our knowledge.

We have pushed an updated version of the Escalate code with fixes to this problem.  Additionally, we have reuploaded correct versions of all affected `statesets` -- this includes crank0022 which is the current crank that has not yet received any recommendations from performers.

A summary of the bug is as follows: During `stateset` generation, second addition of any chemical (in this case acid, and organic) was not being correctly summed with the first addition prior to the calculation of the concentration.  This resulted in inaccurate presentation of the formula concentration associated with particular runs in the `stateset`.  The bug was identified by finding a large number of redundant columns in the `stateset`.

Notes about the fix: The patch does not change the associated labels between experiments in the stateset and the reactions being performed, but information about the experiment was not being accurately expressed in the stateset.  The result when analyzed would make the outcomes appear noisier than expected.

In addition to updating the statesets on both the versioned datarepo and escalation server, we have reached out to consumers / stakeholders in the escalation/escalate to resolve any issues that this mislabeling has caused in their dataset.
